pkg/utils/utils.go

```diff
@@ -16,3 +16,29 @@ func ParseSymbol(address uint64, table *helpers.KernelSymbolTable) *helpers.Kern
 	hookingFunction.Owner = strings.TrimSuffix(hookingFunction.Owner, "]")
 	return hookingFunction
 }
+
+func HasBit(n uint64, offset uint) bool {
```

Comment:
Probably should make all/most of these generic for any numeric type

Reply:
I'll leave generic improvements for further work.